docs/billing.md

```diff
@@ -10,139 +10,171 @@ stack:         all
 
 ---
 
-This article is all about the concepts of billing, pricing, payments, invoices & taxes. Please also check out the [Billing Contact](/billing-contact) article to learn about practical usage in the Dashboard.
+This article covers the topics like billing, pricing, payments, invoices and taxes. Please read the link:/billing-contact[Billing Contact] article to learn about the practical side of things in our Dashboard.
 
 
 ### Free trial
 
-You can test fortrabbit before you pay a dime. All you need to do is: sign up to fortrabbit and try it for free. The trial will end automatically, you don't have to cancel it. No credit card required. Once you decide to book a paid plan the following applies:
+You can test fortrabbit before paying a single a dime, completely for free. All you need to do is sign up and upload some code. The trial will end automatically, you don't have to cancel it and you will not be charged. No credit card required. Once you decide to book a paid plan, the following sections will apply.
 
 
 ### Consumption based billing
 
-Pay after usage: No upfront costs nor payment! **First you book, then you'll use it, then we'll invoice you** for what you have actually used. It's prorate, just like with your old telephone bill. At the end of the month you'll get an invoice for what you have used in that month. 
+There are no upfront costs or payments. The service is priced by usage, similar to how other cloud providers do it. This may break your expectations, but in practice this is how we do things;
 
+1. Customer signs up and opts in to pay for the service
```

Comment:
Please use client instead of customer. https://github.com/fortrabbit/help/#other-words

Reply:
Client ok. I consider 2nd person voice to be less than ideal for technical documentation, also the list is connected to the paragraph above.

Reply:
You need to make a decision. I do not consider "maybe rewrite" as an imperative statement.

Reply:
Your English is way better than mine + you are smarter than I am. So I suggest with a maybe, because maybe there is something I am missing.

Reply:
I changed this so it would fit in with the rest of the document.